codeforamerica/vita-min
# FYST 198 Implement Form AZ-322 XML and PDF

## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-198

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
XML and PDF added if any public school contributions are made. 

## How to test?
Use any az xml to test on heroku. On the page for public school contributions add at least 4 schools in order to see all the pages of the PDF for AZ322 to be filled out and all sections of Form322 xml.
